challenges.yml

```diff
@@ -40,16 +40,15 @@ extending_instancedynamicdata:
 
 extending_singlefilecomponent:
   question: |
-    Imagine our application has a `.vue` single file component that renders as
-    the page title. It has a `title` data member that it uses to render the
+    Imagine you want to build a `.vue` single file component that renders a
+    page header. It needs to have a `title` data key that it uses to render the
     template.
 
-    Which one of these code snippets would compile correctly into such
-    component?
+    Which one of these code snippets would accomplish this?
```

Comment:
compile! Are you trying to scare PHP devs? ;) I softened this to sound less technical

Reply:
LOL